loop-functions/CMakeLists.txt

```diff
@@ -15,3 +15,6 @@ add_subdirectory(vanilla)
 #add_subdirectory(ModularityExperiment)
 #add_subdirectory(visual-add-ons)
 #add_subdirectory(localsearch)
+add_subdirectory(behaviortrees)
```

Comment:
This should not be activated by default, only vanilla should

Reply:
The rest is fine, once you've fixed that, lets merge

Reply:
```suggestion
#add_subdirectory(behaviortrees)
```